operations/conman/Establish_a_Serial_Connection_to_NCNs.md

```diff
@@ -1,120 +1,148 @@
 # Establish a Serial Connection to NCNs
 
 The ConMan pod can be used to establish a serial console connection with each non-compute node (NCN) in the system.
-In the scenario of a power down or reboot of an NCN worker, one must first determine if any `cray-console-node` pods
-are running on that NCN. It is important to move `cray-console-node` pods to other worker nodes before rebooting or
+
+In the scenario of a power down or reboot of an NCN worker, one must first determine if any `cray-console` pods
+are running on that NCN. It is important to move `cray-console` pods to other worker nodes before rebooting or
 powering off a worker node.
 
 If a `cray-console-node` pod is running on a worker node when it is powered off or rebooted, then access to its
 associated consoles will be unavailable until one of the following things happens:
+
 * the worker node comes back up and the `cray-console-node` pod begins running on it.
 * the `cray-console-node` pod is terminated and comes up on another worker node.
 * the `cray-console-operator` pod assigns the associated consoles to a different `cray-console-node` pod.
 
 ## Prerequisites
 
-The user performing this procedure needs to have access permission to the `cray-console-operator` and `cray-console-node` pods.
+The user performing these procedures needs to have access permission to the `cray-console-operator` and `cray-console-node` pods.
 
-## Procedure
+## Connection procedure
 
-> **`NOTE`** this procedure has changed since the CSM 0.9 release.
+1. When trying to monitor a worker node that will be rebooted or powered down, first follow the
+   [Evacuation procedure](#evacuation-procedure).
 
-1. Find the `cray-console-operator` pod.
+1. (`ncn-mw#`) Find the `cray-console-node` pod that is connecting with the console.
 
     ```bash
-    OP_POD=$(kubectl get pods -n services \
-            -o wide|grep cray-console-operator|awk '{print $1}')
-    echo $OP_POD
+    NODE_POD=$(kubectl -n services exec $OP_POD -c cray-console-operator -- sh -c \
```

Comment:
This uses 'OP_POD', but it looks like you removed the directions to get it defined?

Reply:
Fixed